pkg/api-server/inspect_endpoints.go

```diff
@@ -73,14 +74,44 @@ func (s *simpleMatchedPolicyGetter) Get(ctx context.Context, dataplaneKey core_m
 	return &proxy.Policies, nil
 }
 
-func addInspectEndpoints(ws *restful.WebService, mpg core_xds.MatchedPoliciesGetter) {
+var policies = map[core_model.ResourceType]bool{
+	core_mesh.TrafficPermissionType: true,
+	core_mesh.FaultInjectionType:    true,
+	core_mesh.RateLimitType:         true,
+	core_mesh.TrafficLogType:        true,
+	core_mesh.HealthCheckType:       true,
+	core_mesh.CircuitBreakerType:    true,
+	core_mesh.RetryType:             true,
+	core_mesh.TimeoutType:           true,
+	core_mesh.TrafficTraceType:      true,
```

Comment:
Just curiosity - why we are missing
```
ProxyTemplate
TrafficRoute
```
When all others are provided? Is something extra happening with these two policies, so they can not be associated easily with DPPs or is there any different reason?

Reply:
also, since we introduced ResourceDescriptor, I think we are trying to avoid such list of resources, because it's easy to forget to add stuff to this list when you add a new resource. 

Would it make sense to add something like `include_in_inspect` or `allow_to_inspect` to ResourceDescriptor and get rid of this list?

Reply:
Good idea! Added `allow_to_inspect` option